setuptools/command/easy_install.py

```diff
@@ -576,7 +576,7 @@ def check_pth_processing(self):
             _one_liner(
                 """
             import os
-            f = open({ok_file!r}, 'w')
+            f = open({ok_file!r}, 'w', encoding="utf-8")
```

Comment:
Throw-away file only used internally. It should not matter to change its encoding.